examples/application_commands/cog_example.py

```diff
@@ -0,0 +1,32 @@
+from nextcord import (
```

Comment:
Id prefer if we used import nextcord here instead

Reply:
just looks better when using from imports imo

Reply:
I do agree with you @VincentRPS however I find people copying examples with from imports when they don't use it very much then clutters their imports a lot as they don't know what does what

Reply:
I think here it just clutters imports aaand isnt consistent with the rest of the examples